manifest.json

```diff
@@ -1,50 +1,40 @@
 {
-	"name": "Roll Together",
-	"version": "3.0.1",
-	"description": "Synchronize Crunchyroll Videos and Skip Intros",
-	"background": {
-		"scripts": [
-			"background.js"
-		],
-		"persistent": true
-	},
-	"content_scripts": [
-		{
-			"all_frames": true,
-			"matches": [
-				"*://static.crunchyroll.com/*"
-			],
-			"css": [
-				"styles.css"
-			],
-			"js": [
-				"content_script.js"
-			]
-		}
-	],
-	"content_security_policy": "script-src 'self' 'sha256-3Ysffha/L9RFJ1cEzjlxPSHl7yyXbbb05h9eNgaAqgE='; object-src 'self'",
-	"permissions": [
-		"storage",
-		"declarativeContent",
-		"*://www.crunchyroll.com/*",
-		"*://static.crunchyroll.com/*",
-		"*://roll-together-intro-skip.herokuapp.com/*"
-	],
-	"page_action": {
-		"default_popup": "popup.html",
-		"default_icon": {
-			"16": "images/get_started16.png",
-			"32": "images/get_started32.png",
-			"48": "images/get_started48.png",
-			"128": "images/get_started128.png"
-		}
-	},
-	"icons": {
-		"16": "images/get_started16.png",
-		"32": "images/get_started32.png",
-		"48": "images/get_started48.png",
-		"128": "images/get_started128.png"
-	},
-	"options_page": "options.html",
-	"manifest_version": 2
+  "name": "Roll Together",
+  "version": "3.0.1",
+  "description": "Synchronize Crunchyroll Videos",
+  "background": {
+    "service_worker": "service_worker.js",
+    "type": "module"
+  },
+  "content_scripts": [
+    {
+      "all_frames": true,
+      "matches": ["*://static.crunchyroll.com/*"],
+      "js": ["content_script.js"]
+    }
+  ],
+  "permissions": ["storage"],
+  "host_permissions": [
+    "*://www.crunchyroll.com/*",
+    "*://static.crunchyroll.com/*",
+    "*://rt-sync.azurewebsites.net/*"
+  ],
+  "action": {
+    "default_popup": "popup.html",
+    "default_icon": {
+      "16": "images/get_started16.png",
+      "32": "images/get_started32.png",
+      "48": "images/get_started48.png",
+      "128": "images/get_started128.png"
+    }
+  },
+  "icons": {
+    "16": "images/get_started16.png",
+    "32": "images/get_started32.png",
+    "48": "images/get_started48.png",
+    "128": "images/get_started128.png"
+  },
+  "options_page": "options.html",
+  "manifest_version": 3,
+  "minimum_chrome_version": "116"
```

Comment:
Just curious, where did you get this 'minimum_chrome_version' value from?

Reply:
From here: [Chrome 116 introduced the following service worker lifetime improvements](https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/lifecycle#:~:text=Chrome%20116%20introduced%20the%20following%20service%20worker%20lifetime%20improvements%3A)

Basically it allows websocket messages to extend the lifetime of the extension.